src/main/java/seedu/address/logic/commands/csvcommand/LoadCommand.java

```diff
@@ -0,0 +1,174 @@
+package seedu.address.logic.commands.csvcommand;
```

Comment:
ImportCommand instead of LoadCommand? because the other one is ExportCommand

Reply:
I thought import sounded weird lol but can. And also regarding your last comment, yea I did that 👍

Reply:
Oh wait, i dont think i did. lemme change